lib/sarif/language_version/base_sarif.rb

```diff
@@ -0,0 +1,32 @@
+require 'salus/bugsnag'
+
+module Sarif::LanguageVersion
+  class BaseSarif < Sarif::BaseSarif
+    include Salus::SalusBugsnag
+
+    LANGUAGE_VERSION_DOC_URI = "https://github.com/coinbase/salus/blob/master/docs/scanners/"\
+    "language_version_scan.md".freeze
+
+    def initialize(scan_report, repo_path = nil)
+      super(scan_report, {}, repo_path)
+      @uri = LANGUAGE_VERSION_DOC_URI
+      @logs = parse_scan_report!
+    end
+
+    def parse_scan_report!
+      @scan_report.errors
+    end
+
+    def parse_issue(issue)
+      {
+        id: "LV0001",
```

Comment:
Minor nit: Can you store that in a const
LANGUAGE_VERSION_MISMATCH = "LV0001".freeze
or similar